.gitignore

```diff
@@ -24,6 +24,7 @@ website/node_modules
 *.test
 *.iml
 .envrc
+.tool-versions
```

Comment:
What is this for?

Reply:
It's used by `asdf` (https://github.com/asdf-vm/asdf) which we tend to use at GitLab (primarily due to it being used in [GDK](https://gitlab.com/gitlab-org/gitlab-development-kit)) to manage tool versions in our development environments since we have so many different projects.

Reply:
ah cool